src/codegen/utilities/interpreter_helpers.py

```diff
@@ -122,6 +140,41 @@ def get_instantiation_lines_for_output(func):
     return instantiation_lines
 
 
+def get_instantiation_lines_for_varargs(func):
+    """Gets instantiation lines for functions with variable arguments."""
+    instantiation_lines = []
+    if func.function_name in CDLL_EXEC_STYLE_VARARGS_FUNCTIONS:
+        for param in func.output_parameters:
+            instantiation_lines.append(f"{param.parameter_name}_element = {param.ctypes_data_type}")
+            instantiation_lines.append(
+                f"{param.parameter_name}.append({param.parameter_name}_element)"
+            )
+    return instantiation_lines
+
+
+def get_argument_definition_lines_for_varargs(varargs_params):
+    argument_defininion_lines = []
+    for param in varargs_params:
+        argtype = to_param_argtype(param)
+        if param.direction == "in":
+            argument_defininion_lines.append(f"args.append({param.parameter_name}[index])")
+        else:
+            argument_defininion_lines.append(
+                f"args.append(ctypes.byref({param.parameter_name}_element))"
+            )
+        argument_defininion_lines.append(f"argtypes.append({argtype})")
+        argument_defininion_lines.append("")
+    return argument_defininion_lines
+
+
+def get_varargs_parameters(func):
+    varargs_parameters = []
+    if func.function_name in CDLL_EXEC_STYLE_VARARGS_FUNCTIONS:
+        varargs_parameters = func.parameters
+        del varargs_parameters[0]
```

Comment:
The varargs parameters are described in the metadata: https://github.com/ni/nidaqmx-python/blob/master/src/codegen/metadata/functions.py#L12740
- The C parameters have `repeating_argument`
- The gRPC parameters have `repeated_var_args`
- The metadata doesn't really describe the Python parameters

Suggestion:
- Add an `is_repeating_argument` property to the `Parameter` class. Edit: @Sakthi-SM has already done that in https://github.com/ni/nidaqmx-python/pull/285/
- Change this code to `return [p for p in func.parameters if p.is_repeating_argument]`
- Elsewhere, replace `if func.function_name in CDLL_EXEC_STYLE_VARARGS_FUNCTIONS` with `if any(get_varargs_parameters(func))`

Reply:
updated as per suggestion